peachjam/admin.py

```diff
@@ -152,7 +152,21 @@ class SourceFileInline(BaseAttachmentFileInline):
     readonly_fields = (*BaseAttachmentFileInline.readonly_fields, "source_url")
 
 
+class TopicChoiceField(forms.ModelChoiceField):
+    def label_from_instance(self, obj):
+        return f"{'-'*(obj.depth-1)}{obj.name}"
```

Comment:
```suggestion
        return f"{'-'*(obj.depth-1)} {obj.name}"
```

I think it's a touch easier to read with a space between the dashes and the name.